server/migrations/20200104194451-sonoff-to-tasmota.js

```diff
@@ -1,87 +0,0 @@
-const Promise = require('bluebird');
```

Comment:
Why did you delete this migration?

Reply:
Because it uses service model which changes. It was not olny based on SQL but on code too.
This migration is supposed to change all 'sonoff' devices to tasmota (external id, service id).
But I don't thik this is really useful.